ereolen_support/ereolen_material.form.inc

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen material selection form.
+ */
+
+include_once 'ereolen_support.helper.inc';
+
+/**
+ * Create material selection form.
+ *
+ * @param array $form
+ *   The form to create.
+ * @param array $form_state
+ *   The state of the form.
+ *
+ * @return mixed
+ *
+ * @throws \Exception
+ */
+function ereolen_material_form($form, &$form_state) {
+  $form['#attributes'] = [
+    'name' => 'ereolen_material_form',
+    'class' => ['node-webform'],
+    'onsubmit' => 'op.disabled = true; return true;'
+  ];
+
+  $form['#attached']['css'] = [
+    drupal_get_path('module', 'ereolen_support') . '/css/styles.css'
+  ];
+
+  $form['ereolen_material_lead'] = [
+    '#markup' => theme('ereolen_support_lead_text', [
+      'text' =>  variable_get('ereolen_material_lead')['value'] ?? NULL,
+    ]),
+  ];
+
+  $form['ereolen_material_email'] = [
+    '#type' => 'emailfield',
+    '#title' => t('Email'),
+    '#required' => TRUE,
+  ];
+
+  $form['ereolen_material_description'] = [
+    '#type' => 'textarea',
+    '#title' => t('Description'),
+    '#required' => TRUE,
+  ];
+
+  $form['submit_button'] = [
+    '#type' => 'submit',
+    '#value' => t('Send'),
+  ];
+
+  honeypot_add_form_protection($form, $form_state, array('honeypot', 'time_restriction'));
```

Comment:
Use short array notation here (`[…]`) for consistency with the rest of your code.